docs/Model-Adapter.md

```diff
@@ -126,6 +195,27 @@ def self.matches_conditions_hash?(subject, conditions)
 end
 ```
 
-The first one returns `true` when there's a conditions option which is not a Symbol (such as `:age.gt`). The second method will be called by CanCan when the first one returns true to check if the given subject matches the hash of conditions.
+### Additional Examples
+
+Eventhough CanCanCan tries to make the implementation of custom adapters easy and flexible, 
+it can be hard task.
+
+Thus you'd probably be best served with inspecting the actual implementation of the activerecord adapter to get 
+a better overview how a battle tested adapter is structured and implemented.
+
 
-See the actual [[mongoid_adapter_spec.rb|https://github.com/ryanb/cancan/blob/master/spec/cancan/model_adapters/mongoid_adapter_spec.rb]] and [[mongoid_adapter.rb|https://github.com/ryanb/cancan/blob/master/lib/cancan/model_adapters/mongoid_adapter.rb]] files for the full code.
+#### Implementation
+
+
+* [ActiveRecord Base](https://github.com/CanCanCommunity/cancancan/blob/develop/lib/cancan/model_adapters/active_record_adapter.rb)
+* [ActiveRecord 4](https://github.com/CanCanCommunity/cancancan/blob/develop/lib/cancan/model_adapters/active_record_4_adapter.rb)
+* [ActiveRecord 5](https://github.com/CanCanCommunity/cancancan/blob/develop/lib/cancan/model_adapters/active_record_5_adapter.rb)
+
+#### Tests / Specs
+
+* [ActiveRecord Base](https://github.com/CanCanCommunity/cancancan/blob/develop/spec/cancan/model_adapters/active_record_4_adapter_spec.rb)
+* [ActiveRecord 4](https://github.com/CanCanCommunity/cancancan/blob/develop/spec/cancan/model_adapters/active_record_5_adapter_spec.rb)
+* [ActiveRecord 5](https://github.com/CanCanCommunity/cancancan/blob/develop/spec/cancan/model_adapters/active_record_adapter_spec.rb)
```

Comment:
@Liberatys can you point to  `[ActiveRecord Base](./active_record_4_adapter_spec.rb)` for example?

Reply:
👍🏻